Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -216,10 +216,39 @@ def GetCrosshairs(self):
         return actor, vert, horiz
 
     def GetImageWorldExtent(self):
+        """Deprecated. Use `GetDataExtentInWorld` and `GetVoxelsFromExtent`."""
+        return self.image2world(self.GetInputData().GetExtent()[1::2])
+
+    def GetDataExtentInWorld(self):
         """
-        Compute and return the maximum extent of the image in the rendered world
+        Compute and return the extent of the input data in the rendered world.
         """
-        return self.image2world(self.GetInputData().GetExtent()[1::2])
+        data_extent_image = self.GetInputData().GetExtent()
+        data_extent_world = self.image2worldExtent(data_extent_image)
+        return data_extent_world
+
+    def GetVoxelsFromExtent(self, extent):
+        """Given the extent of a box or image, gets the voxels corresponding to the min values in all directions
+        and max values in all directions."""
+        voxel_min = extent[0::2]
+        voxel_max = extent[1::2]
+        return voxel_min, voxel_max
+
+    def image2worldExtent(self, extent):
+        """Given the extent of a box or image, gets the voxels corresponding to the min values in all directions
+        and max values in all directions. Then, converts their coordinates in the world coordinate system. 
+        Returns the converted extent."""
+        v1_image, v2_image = self.GetVoxelsFromExtent(extent)
+        v1_world = self.image2world(v1_image)
+        v2_world = self.image2world(v2_image)
+        extent_world = self.GetExtentFromVoxels(v1_world, v2_world)
```

Comment:
I don't like too much `GetVoxelsFromExtent` and its pair `GetExtentFromVoxels`, but it's ok.

Reply:
I cannot think of a better name, I will leave it for now